### PR TITLE
#8Task. valery.aleshka

### PR DESCRIPTION
**Pull request by Valeryaleshka**
https://travis-ci.org/github/Valeryaleshka/js-assignments/builds/715786789

### CODE REVIEW

File: task/08-objects-tasks.js [hunk 1]
```diff
@@ -23,7 +23,17 @@
...
-    throw new Error('Not implemented');
+
+    this.width = width;   
+    this.height = height;
+
+    Rectangle.prototype.getArea = function () {
+        return this.width * this.height;
+    }
+
+    return this;
```
Comment (KostyaKov) [inline task/08-objects-tasks.js]: "return" is redundant here

File: task/08-objects-tasks.js [hunk 2]
```diff
@@ -38,7 +48,10 @@ function Rectangle(width, height) {
  *    { width: 10, height : 20 } => '{"height":10,"width":20}'
  */
 function getJSON(obj) {
-    throw new Error('Not implemented');
+
+    return JSON.stringify(obj);
+
+    //throw new Error('Not implemented');
```
Comment (KostyaKov) [inline task/08-objects-tasks.js]: It is better to delete unnecessary comments

**Review by KostyaKov — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Valeryaleshka** [comment]
I've fixed everything.
New Travis-ci build.
https://travis-ci.org/github/Valeryaleshka/js-assignments/builds/721846214

**Review by KostyaKov — COMMENTED** [review]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**